src/app/screens/Settings.tsx

```diff
@@ -34,6 +46,17 @@ function Settings() {
   const [cameraPermissionsGranted, setCameraPermissionsGranted] =
     useState(false);
 
+  function closeModal() {
+    setModalIsOpen(false);
+  }
+
+  async function updateAccountPassword(password: string) {
+    await utils.call("changePassword", {
+      password: formData.password,
+    });
+    closeModal();
```

Comment:
I think adding a toast message her with a confirmation that the password was changed would be good. 
To make sure the user is not wondering if it worked or not.